docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -133,7 +131,7 @@ _ Support for Windows Hello, conditional access policies, and FIDO keys.
 _ Streamlined single sign-on.
 _ Bug fixes and enhancements shipped with Windows.
 
-Signing in with WAM is a preview, opt-in feature. Once enabled, it will replace the previous browser-based user interface. 
+Signing in with WAM is a preview, opt-in feature. Once enabled, replaces the previous browser-based user interface. 
```

Comment:
consider `Once enabled, the previous browser-based user interface is replaced.`  `replaces` doesn't make sense by itself.